app/src/main/java/org/jellyfin/androidtv/ui/playback/VideoManager.java

```diff
@@ -98,17 +98,22 @@ public VideoManager(PlaybackOverlayActivity activity, View view) {
         mSubtitlesSurface.setZOrderMediaOverlay(true);
         mSubtitlesSurface.getHolder().setFormat(PixelFormat.TRANSLUCENT);
 
-        mExoPlayer = new ExoPlayer.Builder(TvApp.getApplication(), new DefaultRenderersFactory(TvApp.getApplication()) {
+        ExoPlayer.Builder exoPlayerBuilder = new ExoPlayer.Builder(activity);
+        exoPlayerBuilder.setRenderersFactory(new DefaultRenderersFactory(activity) {
             @Override
             protected void buildTextRenderers(Context context, TextOutput output, Looper outputLooper, int extensionRendererMode, ArrayList<Renderer> out) {
                 // Do not add text renderers since we handle subtitles
             }
-        }).build();
+        });
+
+        DataSource.Factory baseDataSourceFactory = new DefaultHttpDataSource.Factory().setUserAgent("ATV/ExoPlayer");
```

Comment:
If we're adding a user agent here we should probably add the app version too.

Reply:
I only included the data source factory stuff because the removed `DefaultDataSourceFactory` used a user agent. If it's unneeded I'd rather just remove the whole thing.

Otherwise I can look into adding the app version to the user agent

Reply:
I don't think we really need to set the user agent. The SDK doesn't set one either (although the http library might set one).